Dockerfile

```
@@ -123,11 +126,14 @@ RUN bash -lc "gem update --system && gem install bundler"
 
 # install conan
 RUN apt-get install -y python-dev && \
-	pip install --ignore-installed six --ignore-installed colorama --ignore-installed requests --ignore-installed chardet --ignore-installed urllib3 --upgrade setuptools && \
-    pip install -Iv conan==1.11.2
+	pip install --no-cache --ignore-installed six --ignore-installed colorama \
+	    --ignore-installed requests --ignore-installed chardet \
+	    --ignore-installed urllib3 \
+	    --upgrade setuptools && \
+    pip install --no-cache -Iv conan==1.11.2
```

@bluefish6 Did you also mean `--no-cache-dir` here as well?

---

Yeah, sorry, my bad.

```suggestion
    pip install --no-cache-dir -Iv conan==1.11.2
```